src/main/kotlin/com/criptext/mail/scenes/composer/ComposerController.kt

```diff
@@ -138,6 +138,7 @@ class ComposerController(private val storage: KeyValueStorage,
             model.filesSize -= file.length()
             model.attachments.removeAt(position)
             scene.notifyAttachmentSetChanged()
+            model.attachmentsChanged = true
```

Comment:
```suggestion
            model.attachmentsChanged = model.attachments.isNotEmpty()
```
This is in case all the attachments are removed, the draft should not be saved.